src/CasCap.Common.Caching/Models/CachingOptions.cs

```diff
@@ -15,4 +15,6 @@ public class CachingOptions
 #pragma warning restore CS8618 // Non-nullable field is uninitialized. Consider declaring as nullable.
     public string ChannelName { get; set; } = "expiration";
     public int MemoryCacheSizeLimit { get; set; }
-}
\ No newline at end of file
+    public int KeyedLockPoolSize { get; set; } = 20;
```

Comment:
These two properties are not specific to the Caching library, they are only pertinent to your library and so these should be loaded by configuration in your library.

Reply:
There currently isn't a means to do this; they're passed on as parameters. I put them in your configuration to avoid hardcoding them. As far as I am aware, people are happy with these default values of 20, initial fill 1 and aren't changing them, so they're generally hardcoded. Would you prefer that?

Reply:
Made its own configuration.